packages/payment/src/services/payment-module.ts

```diff
@@ -95,13 +100,24 @@ export default class PaymentModuleService<
     this.captureService_ = captureService
     this.paymentService_ = paymentService
     this.paymentSessionService_ = paymentSessionService
+    this.paymentProviderService_ = paymentProviderService
     this.paymentCollectionService_ = paymentCollectionService
   }
 
   __joinerConfig(): ModuleJoinerConfig {
     return joinerConfig
   }
 
+  __hooks = {
+    onApplicationStart: async () => await this.createProvidersOnLoad(),
+  }
+
+  /**
+   *
+   * ********** PAYMENT COLLECTION **********
+   *
+   */
+
```

Comment:
todo: pluralize methods that are bulk, it is part of the convention document, I let you have a look